data/migrations/20200129182841_users.js

```diff
@@ -0,0 +1,47 @@
+exports.up = async function(knex) {
+  await knex.schema.createTable('users', table => {
+    table.increments('id');
```

Comment:
not an issue, but 'id' is default value if nothing is passed in

Reply:
Ah ha! Good to know. I thought I needed the name for referencing in my other table.